random_menu.rb

```diff
@@ -0,0 +1,19 @@
+# Random Menu Generator for Boozy Desserts (Lunch) - Loraine Kanervisto
+# Thanks Pinterest for the inspiration
+
+#Creates an adjective array
+adjs = ["salty", "buttery", "spicy", "creamy", "frosty", "bitter", "sour", "sweet", "chunky", "hot", "cheesy", "burnt", "candied", "drizzled", "fluffy"]
+
+#Creates a booze array
+booze = ["champagne", "stout", "rum", "vodka", "moonshine", "Kahlua", "bourbon", "red wine", "Bacardi 151", "tequila", "malt liquor", "Amaretto", "peach schnapps", "cider", "Fireball"]
+
+#Creates a dessert array
+dessert = ["coffee", "shake", "ice cream", "brownie", "waffle", "cake", "jello", "cookie", "doughnut", "pudding", "eclair", "cream puff", "pastry", "pie", "candy" ]
+
+#Loop to print numbered list
+list_num = 1
+# This conditional allows the list to automatically adjust to the count of the adjective array.
+while list_num <= adjs.count
```

Comment:
I like how you used .count instead of 15.  .count works, and .length or .size or work, too. I think in this case maybe .length or .size might be preferred, but it works, so this is good for now
(http://batsov.com/articles/2014/02/17/the-elements-of-style-in-ruby-number-13-length-vs-size-vs-count/)